libc/test/UnitTest/CMakeLists.txt

```diff
@@ -111,8 +111,10 @@ add_header_library(
 add_unittest_framework_library(
   LibcFPTestHelpers
   SRCS
+    FEnvSafeTest.cpp
```

Comment:
to keep the bazel build working as well, these files should be added to the `fp_test_helpers` target in `utils/bazel/llvm-project-overlay/libc/test/UnitTest/BUILD.bazel`